lib/src/back_end/code_writer.dart

```diff
@@ -172,6 +172,8 @@ class CodeWriter {
   /// piece to [indent], relative to the indentation of the surrounding piece.
   ///
   /// Replaces any previous indentation set by this piece.
+  ///
+  /// TODO(tall): Add another API that adds/subtracts existing indentation.
```

Comment:
Tiny nit: "///" -> "//" since the TODO isn't really part of the doc comment.

Reply:
Oh yes, thanks for catching that. Fixed.